core/migration_external_user_id.go

```diff
@@ -0,0 +1,134 @@
+// Copyright 2022 Board of Trustees of the University of Illinois.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package core
+
+import (
+	"fmt"
+	"log"
+)
+
+// MigrateScoreExternalUserIDs migrates all score records to populate external_user_id with AmgUUID from Core BB
+func (app *Application) MigrateScoreExternalUserIDs(orgID string, appID string, batchSize int) error {
+	log.Println("Starting Score External User ID migration...")
+	log.Printf("Parameters: orgID=%s, appID=%s, batchSize=%d", orgID, appID, batchSize)
+
+	// Validate batch size
+	if batchSize <= 0 {
+		batchSize = 100 // Default to 100 if invalid
```

Comment:
Let's make this 100 default value a const at the top of this file and use it here as well as in apis_admin.go in your request handler. This way the value will be consistent in both places.